README.md

```diff
@@ -5,7 +5,7 @@ This repository provides OpenTracing instrumentation for Spring Boot and its var
 compatible implementation.
 
 It contains auto-configurations which instruments and trace following Spring Boot projects:
-* Spring Web (RestControllers, RestTemplates, WebAsyncTask)
+* Spring Web (RestControllers, RestTemplates, WebAsyncTask, WebClient, WebFlux)
```

Comment:
Shouldn't this PR bump ot-spring-web version which includes webClient and WebFlux instrumentatiuons

Reply:
Thank you, I didn't realize that it wasn't already bumped. I bumped it to 1.1.1 (latest) now.